targets/kde

```diff
@@ -9,7 +9,12 @@ CHROOTBIN='crouton-noroot startkde'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-install --minimal kde-plasma-desktop -- network-manager
+if release -lt vivid then
+    PACKNAME='kde-plasma-desktop'
```

Comment:
Yeah, and the usage of the variable below should be quoted.  I was going to let some of the style nits slide for this target, but since you brought them up...